bot.py

```diff
@@ -39,6 +39,30 @@ async def popcorn(interaction: nextcord.Interaction):
         response += f"but {kernels} of them are just kernels!"
     await interaction.response.send_message(response)
 
+# Help Command
```

Comment:
Please reformat message response so that after each command and each description there is a new line. Something like:

/watchlist_create

create a new watchlist
/watchlist_delete

delete an existing watchlist
this would require a new line after the command and then 2 new lines after the description